### DISCUSSION TITLE
does not contain package github.com/chrislusf/seaweedfs/go/weed

### DISCUSSION THREAD

**Discussion by joalmeida** [seaweedfs/seaweedfs#2082] [Q&A]
Hello,

root@weed01:/home/user# uname -a
Linux weed01 4.19.0-14-amd64 #1 SMP Debian 4.19.171-2 (2021-01-30) x86_64 GNU/Linux

root@weed01:/home/user# go version
go version go1.16.4 linux/amd64

root@weed01:/home/user# go get github.com/chrislusf/seaweedfs/go/weed
go get: module github.com/chrislusf/seaweedfs@upgrade found (v0.0.0-20210524105044-9079ce0e7f9e), but does not contain package github.com/chrislusf/seaweedfs/go/weed

So, how do I install seaweedfs ?

Best Regards.

**Comment by joalmeida** [comment]
Hello again,

root@weed01:/home/user#
root@weed01:/home/user# wget https://dl.bintray.com/chrislusf/seaweedfs/go_0.70_amd64.deb
--2021-05-24 15:14:53--  https://dl.bintray.com/chrislusf/seaweedfs/go_0.70_amd64.deb
A resolver dl.bintray.com (dl.bintray.com)...18.196.33.98, 3.127.93.119
A ligar a dl.bintray.com (dl.bintray.com)|18.196.33.98|:443... ligado.
Pedido HTTP enviado, a aguardar resposta...403 Forbidden
2021-05-24 15:14:53 ERRO 403: Forbidden.

Direct download doesn't work 


Best Regards

**Reply by chrislusf** [reply]
https://github.com/chrislusf/seaweedfs/releases

**Reply by joalmeida** [reply]
Can You review the step3 of the installation guide in 

http://seaweedfs.github.io/

Best Regards.

**Comment by chrislusf** [comment]
go get github.com/chrislusf/seaweedfs/weed

**Reply by joalmeida** [reply]
```
root@weed01:/home/user# go get github.com/chrislusf/seaweedfs/weed
go: downloading github.com/chrislusf/raft v1.0.7
go: downloading github.com/golang/protobuf v1.4.2

... ( lots of lines, no error  ) ...

go: downloading github.com/googleapis/gax-go v1.0.3
go: downloading github.com/pierrec/lz4 v1.0.1
go: downloading github.com/willf/bloom v1.0.0
go: downloading go.etcd.io/etcd v0.5.0-alpha.5
# github.com/coreos/etcd/clientv3/balancer/picker
/root/go/pkg/mod/github.com/coreos/etcd@v3.3.25+incompatible/clientv3/balancer/picker/err.go:37:44: undefined: balancer.PickOptions
/root/go/pkg/mod/github.com/coreos/etcd@v3.3.25+incompatible/clientv3/balancer/picker/roundrobin_balanced.go:55:54: undefined: balancer.PickOptions
# github.com/coreos/etcd/clientv3/balancer/resolver/endpoint
/root/go/pkg/mod/github.com/coreos/etcd@v3.3.25+incompatible/clientv3/balancer/resolver/endpoint/endpoint.go:114:78: undefined: resolver.BuildOption
/root/go/pkg/mod/github.com/coreos/etcd@v3.3.25+incompatible/clientv3/balancer/resolver/endpoint/endpoint.go:182:31: undefined: resolver.ResolveNowOption
# github.com/DataDog/zstd
cgo: exec gcc: exec: "gcc": executable file not found in $PATH
```

Is Gcc required ? 

Best Regards.

**Reply by chrislusf** [reply]
No

**Reply by joalmeida** [reply]
It's a fresh Debian 10.6.0-amd64 install.
what's missing?

Best Regards.

**Reply by chrislusf** [reply]
```
   34  mkdir -p $HOME/go/src/github.com/chrislusf
   35  cd $HOME/go/src/github.com/chrislusf
   40  apt-get install git
   41  git clone https://github.com/chrislusf/seaweedfs
   42  cd seaweedfs/weed/
   43  go install
   44  weed
```
